src/Microsoft.Azure.ServiceBus/Extensions/MessageInterOpExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Extensions
```

Comment:
hi Sean,

We want to encourage the app to handle the serialization instead of service bus client library. So that people can do necessary optimization and any customization. e.g. nowadays, people might prefer json while performance is not a particular concern. However, we are also skeptically whether we actually should provide some easy way to do so. So this is a starting point to at least support inter-op with old .Net library's data contract serialization.